tenzir/integration/reference/parse-with-grok/step_00.ref

```diff
@@ -0,0 +1,5 @@
+{
+  "version": {
+    "version": "v4.5.0-71-gae887a0ca3-dirty"
```

Comment:
Won't this fail in future versions and only work with this particular commit?

Reply:
There's a `replace version` after the `show version`, there, just so this test is reproducible.

Reply:
Ah, I was just curious because I'm commenting on a checked-in baseline step.